lib/cartopy/mpl/geoaxes.py

```diff
@@ -1161,8 +1161,8 @@ def imshow(self, img, *args, **kwargs):
 #            result.set_clip_path(self.outline_patch)
         return result
 
-    def gridlines(self, crs=None, draw_labels=False, xlocs=None,
-                  ylocs=None, **kwargs):
+    def gridlines(self, crs=None, draw_labels=False, mlocs=None,
```

Comment:
This is part of the public API, so please don't change the names, or you'll have to add some deprecations, etc.